roles/netsocbot/tasks/consul_conf.yml

```diff
@@ -0,0 +1,56 @@
+---
+- name: "Ensure Consul Bot token"
+  run_once: true
+  consul_acl_prefix:
+    scheme: https
+    host: consul.{{ dev_site }}
+    port: 443
+    name: Discord Bot Token
+    token_type: management
+    token: "{{ consul_bot_token }}"
+    mgmt_token: "{{ consul_master_token }}"
+    rules:
+      - key_prefix: "discord/"
+        policy: read
+
+- name: "Ensure Consul KVs"
+  consul_kv:
+    scheme: https
+    host: consul.{{ dev_site }}
+    port: 443
+    key: "discordbot/{{ item.key }}"
+    value: "{{ item.value | to_json }}"
+    token: "{{ consul_master_token }}"
+  with_items:
+    - {
+        key: channels,
+        value:
+          {
+            "public_announcements": "{{ public_announcements }}",
+            "private_events": "{{ private_events }}",
+            "public_general": "{{ public_general }}",
+          },
+      }
+    - {
+        key: servers,
+        value:
+          {
+            "public": "{{ public_server }}",
+            "committee": "{{ committee_server }}",
+          },
+      }
+    - {
+        key: welcome_messages,
+        value: ["Welcome to the Netsoc Discord server %s! :)"],
+      }
+    - {
```

Comment:
could this go back to single lines maybe, this formatting is pretty painful to read imo :P